tests/integration/models/test_model_forward.py

```
@@ -356,6 +356,46 @@ def test_attention_backends_match(self, seq_lengths):
 
 
+@requires_cuda
+class TestNormOutputParams:
+    """Tests for Eagle 3.1: norm_before_fc + norm_output."""
+
+    def test_norm_output(self):
+        model = make_eagle3_model(norm_before_fc=True, norm_output=True)
+        assert model.input_norm is not None
+        samples = _make_samples([128])
+        batch = make_batch(max_len=MAX_LEN, samples=samples, hidden_size=HIDDEN_SIZE)
+        draft_tokens, loss, metrics = model(**batch, ttt_steps=3)
```

Ruff (0.15.18)

[warning] Unpacked variable `metrics` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)